Exercise_C.js

```diff
@@ -14,5 +14,9 @@ See README.md if you need help running your code.
 */
 
 for (let i = 1; i <= 30; i++) {
-  // CODE HERE
+  if(i%3==0,i%5==0){console.log("Hong Kong");}
+    else if (i%3==0){console.log("Hong");}
+    else if (i%5==0){console.log("Kong");}
+    else console.log(i)
+
 }

```

Comment:
Great work
This code runs well and is doing what we ask!
The only thing that I might alter is wrapping the forloop in a function so you can invoke it when you want (you could even dynamically change the maximum value of i.